core/templates/pages/header_css_libs.html

```diff
@@ -2,7 +2,9 @@
 <link rel="preload" href="/third_party/generated/webfonts/fa-brands-400.woff2" as="font" type="font/woff2" crossorigin>
 
 <link rel="stylesheet" type="text/css" media="screen"
-      href="https://fonts.googleapis.com/css?family=Capriola|Roboto|Noto+Serif:ital@0;1|Material+Icons&display=swap">
+      href="https://fonts.googleapis.com/css2?family=Capriola&family=Roboto:wght@400;500;700&family=Noto+Serif:ital@0;1&display=swap">
+<link rel="stylesheet" type="text/css" media="screen"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
Should we still use `&display=swap`?

Reply:
Not sure, so I'll just add it in to avoid surprises.